docs/current_docs/integrations/kubernetes.mdx

```diff
@@ -67,7 +67,7 @@ The previous pattern makes it possible to scale the Dagger deployment, but comes
 
 In this architecture:
 
-- A shared cloud cache stores data from all Dagger Engines running in the cluster. This diagram below references Dagger's experimental Cloud Cache service.
+- A shared cloud cache stores data from all Dagger Engines running in the cluster.
```

Comment:
I see the diagram is updated but the whole "cloud cache" circle should be removed

Reply:
Is this pattern invalid without Dagger Cloud? ie. a user cannot create and drop in their own homebrew cloud cache solution?